src/thunks/fetchCities.js

```diff
@@ -0,0 +1,21 @@
+import { hasError, storeCities } from '../actions';
+import { fetchOptionsCreator } from '../utils/fetchOptionsCreator';
+
+export const fetchCities = () => {
+  return async (dispatch) => {
+    const url = `${process.env.REACT_APP_BACKEND_URL}/graphql`
+    try {
+      const body = '{ allCities{name} }'
```

Comment:
hey @jessicalyn, would it be possible to change the body to 
```
{"query": "{ allCities{name}}"}
```
?
I think with graphql it needs to have the "query" in it

Reply:
Nevermind, I just saw fetchOptionsCreator.js where the query part lives